src/capitalsFirst.js

```diff
@@ -19,6 +19,19 @@
  */
 function capitalsFirst(str) {
   // write code here
-}
+  let arr = str.split(' ');
+  let arrUpperCase = [];
+  let arrLowerCase = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i][0] >= 'A' && arr[i][0] <= 'Z') {
```

Comment:
If you use the same array/object property more then once -> move it to constant
(About arr[i], arr[i][0])